recipe/meta.yaml

```diff
@@ -12,14 +12,16 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
```

Comment:
You need to deprioritize the cuda builds by setting the build number lower than non-cuda builds. (And also add `track_features` to cuda builds

Reply:
@isuruf Can you point me to an example ? I have not used `track_features` before.

Reply:
https://github.com/conda-forge/tensorflow-feedstock/blob/2fdde9bd0977c9356ad192d684c8ca2751bf94ab/recipe/meta.yaml#L128C1-L130 (This one does the opposite of what we should do here)

Reply:
See https://github.com/conda-forge/sympy-feedstock/blob/9b12efb2bf80ddadaf182773f37cf9a6996bd463/recipe/meta.yaml#L8-L10 for how to set different build numbers

Reply:
Shouldn't the run requirement of `__cuda` serve the purpose of allowing non-GPU users to get a non-GPU build by default? If someone has a the CUDA driver I don't understand why we'd want for them to get a non-GPU enabled build by default and have to opt in to get the GPU enabled build?

Reply:
Because this brings in a 1GB package (cudatoolkit) and for some use-cases, I want libhwloc only for CPU use-cases and not for any GPU related work even though I have a GPU.

For eg: pocl has a CPU implementation which needs libhwloc and a GPU implementation which does not need libhwloc.

Reply:
This will be *MUCH* better once we can just use separately packaged libraries like `cudart`.